portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACTests.macro

```diff
@@ -0,0 +1,31 @@
+definition {
+
+	macro deleteABTest {
+		var key_testName = "${testName}";
+
+		Click(locator1 = "ACTests#AB_TEST_NAME");
```

Comment:
Hey @joshitagaki, I think you could use:

`ACNavigation.openItem(itemName = "${testName}");`

Reply:
Oh nice yes that would work. Thanks!